docs/en/operations/system-tables/system_warnings.md

```diff
@@ -18,12 +18,12 @@ If current value drops below the threshold, the entry is removed from the table.
 
 The table can be configured with these settings:
 
-- [max_table_num_to_warn](../server-configuration-parameters/settings.md#max_table_num_to_warn)
-- [max_database_num_to_warn](../server-configuration-parameters/settings.md#max_database_num_to_warn)
-- [max_dictionary_num_to_warn](../server-configuration-parameters/settings.md#max_dictionary_num_to_warn)
-- [max_view_num_to_warn](../server-configuration-parameters/settings.md#max_view_num_to_warn)
-- [max_part_num_to_warn](../server-configuration-parameters/settings.md#max_part_num_to_warn)
-- [max_pending_mutations_to_warn](../server-configuration-parameters/settings.md#max_pending_mutations_to_warn)
+- [max_table_num_to_warn-](../server-configuration-parameters/settings.md#max_table_num_to_warn-max_table_num_to_warn)
```

Comment:
These changed links being you to the exact section of the setting in the doc. For example: https://github.com/ClickHouse/ClickHouse/blob/435dc7483b4d054ef1866f7c9ccd53b5f57937f5/docs/en/operations/server-configuration-parameters/settings.md#max_table_num_to_warn-max_table_num_to_warn 

But doc checks fail :(

The old links seem to make the CI happy, but they don't bring you to the correct section if you click on them.

Reply:
Docs check for the docs build, which is what's generated for clickhouse.com/docs. You are using Github's markdown browser, which is probably secondary at best

Reply:
I didn't know that, TIL.